pkg/machine/qemu/config.go

```diff
@@ -21,18 +21,6 @@ type QEMUVirtualization struct {
 	machine.Virtualization
 }
 
-// setVMFile creates a new machine file pointing to the given path and assigns
```

Comment:
You can just remove the other commit instead.

Reply:
I screwed things up trying to drop commits and such, and just ended up squashing everything into a single commit